docs/design/autoware-interfaces/ad-api/release.md

```diff
@@ -1,5 +1,14 @@
 # Release notes
 
+## v1.4.0
+
+- [New] Add {{ link_ad_api('/api/vehicle/status') }}
```

Comment:
Also `float32 energy_percentage` is being removed, it is a breaking change, it should be in the release notes.

Reply:
This API was first released in v1.4.0 so it is not a breaking change.

Reply:
Alright thanks.